clientv3/README.md

```diff
@@ -72,6 +72,14 @@ if err != nil {
 }
 ```
 
+## Metrics
+
+etcd client provides metrics reporting. This can enable the operators to know how clients are performing and which client generates the most load.
```

Comment:
@gyuho i don't have issues with keeping everything in one line. But can we also keep the info about use for metrics? Since some may be new users and providing the use of metrics can be useful to them? What do you feel?